src/api/core/tokenManager.ts

```diff
@@ -266,4 +268,11 @@ export class TokenManager {
   private get refreshUrl(): string {
     return `${this.url}/${API.REFRESH}`;
   }
+
+  private errorMessage({ httpStatus: { code, status } }: IRequestError): string {
```

Comment:
What do you think of making this method `public` and renaming it to `getErrorMessage`? this way in the authorization failed you can test if the method was called, and you can add isolated tests for this method.

Reply:
Done!